src/main/java/net/mcreator/element/ModElementTypeLoader.java

```diff
@@ -24,99 +24,110 @@
-						Particle.class));
+						Particle.class), true);
 		ModElementType.PLANT = register(
-				new ModElementType<>("plant", 'n', BaseType.BLOCK, RecipeType.BLOCK, PlantGUI::new, Plant.class));
+				new ModElementType<>("plant", 'n', BaseType.BLOCK, RecipeType.BLOCK, PlantGUI::new, Plant.class), true);
 		ModElementType.POTION = register(
-				new ModElementType<>("potion", 'z', BaseType.OTHER, RecipeType.NONE, PotionGUI::new, Potion.class));
+				new ModElementType<>("potion", 'z', BaseType.OTHER, RecipeType.NONE, PotionGUI::new, Potion.class), true);
 		ModElementType.POTIONEFFECT = register(
 				new ModElementType<>("potioneffect", null, BaseType.OTHER, RecipeType.NONE, PotionEffectGUI::new,
-						PotionEffect.class));
+						PotionEffect.class), true);
 		ModElementType.PROCEDURE = register(
 				new ModElementType<>("procedure", 'p', BaseType.OTHER, RecipeType.NONE, ProcedureGUI::new,
-						Procedure.class));
+						Procedure.class), true);
 		ModElementType.RANGEDITEM = register(
 				new ModElementType<>("rangeditem", 'q', BaseType.ITEM, RecipeType.ITEM, RangedItemGUI::new,
-						RangedItem.class));
+						RangedItem.class), true);
 		ModElementType.RECIPE = register(
-				new ModElementType<>("recipe", 'r', BaseType.DATAPACK, RecipeType.NONE, RecipeGUI::new, Recipe.class));
+				new ModElementType<>("recipe", 'r', BaseType.DATAPACK, RecipeType.NONE, RecipeGUI::new, Recipe.class), true);
 		ModElementType.STRUCTURE = register(
 				new ModElementType<>("structure", 's', BaseType.FEATURE, RecipeType.NONE, StructureGenGUI::new,
-						Structure.class));
+						Structure.class), true);
 		ModElementType.TAB = register(
-				new ModElementType<>("tab", 'w', BaseType.OTHER, RecipeType.NONE, TabGUI::new, Tab.class));
+				new ModElementType<>("tab", 'w', BaseType.OTHER, RecipeType.NONE, TabGUI::new, Tab.class), true);
 		ModElementType.TAG = register(
-				new ModElementType<>("tag", 'j', BaseType.DATAPACK, RecipeType.NONE, TagGUI::new, Tag.class));
+				new ModElementType<>("tag", 'j', BaseType.DATAPACK, RecipeType.NONE, TagGUI::new, Tag.class), true);
 		ModElementType.TOOL = register(
-				new ModElementType<>("tool", 't', BaseType.ITEM, RecipeType.ITEM, ToolGUI::new, Tool.class));
+				new ModElementType<>("tool", 't', BaseType.ITEM, RecipeType.ITEM, ToolGUI::new, Tool.class), true);
 
 		// Unregistered type used to mask legacy removed mod element types
 		ModElementType.UNKNOWN = new ModElementType<>("unknown", null, BaseType.OTHER, RecipeType.NONE,
 				(mc, me, e) -> null, GeneratableElement.Unknown.class);
 	}
 
-	private static ModElementType<?> register(ModElementType<?> elementType) {
+	/**
+	 * <p>Registers a {@link ModElementType} by adding it to the registry list.</p>
+	 *
+	 * @param elementType <p>The {@link ModElementType} to add to the registry list</p>
+	 * @param addToModWT <p>When true, this {@link ModElementType} will be added to the Mod {@link net.mcreator.workspace.types.WorkspaceType}</p>
```

Comment:
>_addToModWT:_ When true, this `ModElementType` will be added to the Mod `WorkspaceType`

Then, as it now seems to always be _true_, can you give an example of a MET not supported by this WT?

Reply:
For METs that are not made yet.